dts/arm/nordic/nrf51822.dtsi

```diff
@@ -1,6 +1,7 @@
 #include <arm/armv6-m.dtsi>
 #include <dt-bindings/i2c/i2c.h>
 #include <dt-bindings/gpio/gpio.h>
+#include <nordic/nrf5.dtsi>
```

Comment:
`#include "nrf5.dtsi"` would be more adequate. And I think the file should be named "nrf5_common.dtsi".

Reply:
Was done to match nrf51822_qfaa.dtsi, etc, files. But using "nrf5.dtsi" now.

Reply:
Changed name to nrf_common.dtsi, changed include to be as stated (just with the new name).